internal/services/delegation.go

```diff
@@ -119,8 +119,11 @@ func (s *Service) processBTCDelegationUnbondedEarlyEvent(
 		return err
 	}
 
+	// TODO: save timelock expire, need to figure out what will be the expire height in this case.
+	// https://github.com/babylonlabs-io/babylon-staking-indexer/issues/28
```

Comment:
I think after https://github.com/babylonlabs-io/babylon/pull/215/, Babylon can emit the early unbonding event with the height

Reply:
tracked in https://github.com/babylonlabs-io/babylon/issues/225. Will address it before devnet 5